fractions/microprofile/microprofile-jwt/src/main/java/org/wildfly/swarm/microprofile/jwtauth/MicroProfileJWTAuthFraction.java

```diff
@@ -103,6 +103,19 @@
     @AttributeDocumentation("Roles properties map")
     private Map<String, String> rolesPropertiesMap;
 
+    @AttributeDocumentation("If both this and 'thorntail.microprofile.jwt.realm' properties are enabled then the @LoginConfig annotation does not have to be present."
+                            + "If this option is disabled then the MP JWT authentication mechanism will not be activated")
+    @Configurable("thorntail.microprofile.jwt.enabled")
+    private Defaultable<Boolean> jwtEnabled = bool(true);
```

Comment:
So, do we really need this? It's enabled by default, so users don't have to enable it explicitly. And if they include the MP JWT fraction, they probably want MP JWT :-)

If we want this, I'd rephrase the documentation. Something like:

> Set this to false to disable the MP JWT authentication mechanism. Defaults to true.

And if you want to mention that `@LoginConfig` is not necessary, I'd put it to the documentation of `jwtRealm`, which already mentions that if set, the `realmName` property of the `@LoginConfig` annotation is optional.

Reply:
@Ladicek I'm not 100% sure now myself if it is needed. It can be handy I guess if a user wants to ensure it is not activated against random `Application` without `LoginConfig`, as an extra safety switch, or, just disable it anyway as Quarkus can do, so on balance I guess it should be OK to keep it.

Reply:
@Ladicek have a look please at the updated docs, the hardest part as usual for me :-)